src/api.rs

```diff
@@ -79,6 +79,12 @@ pub struct BlockTime {
     pub height: u32,
 }
 
+#[derive(Debug, Clone, serde::Deserialize)]
+pub struct Block {
```

Comment:
Can we name this differently so that it doesn't shadow `bitcoin::Block`?

Reply:
ACK.. Will do if we finalize on this..